### ISSUE TITLE
Compilation error on clean startup

### ISSUE THREAD

**Issue by KayhanB** [bluelibs/bluelibs#158]
**Compilation error on clean startup**
I want to try your awesome framework for the first time but got an error on the startup.

Tried with node versions 14, 16, 17

**Screenshots**
![Screenshot0](https://user-images.githubusercontent.com/29417376/149621310-1574e08b-3491-4db4-baf5-c0efd1138e56.png)

**Desktop Enviroment:**
 - OS: macOS Monterey
 - CPU: Apple M1 chip

**Comment by theodorDiaconu** [comment]
Yes, this is related to mongodb latest version which introduced this ridiculous recursiveness to aid nested fields. Try using mongodb 4.2.x instead, let me know if this fixes for you. A long term solution is on the way, I am working with mongo team on this one.

**Comment by theodorDiaconu** [comment]
I've fixed the `@bluelibs/x` and also ensured mongo-bundle requires 4.2.2 as a version. If you go through the process again it should be seamless again, thank you for pointing it out.

I estimate a long-term fix in the next 1 month, I'm already following-up with them here: https://www.mongodb.com/community/forums/t/mongodb-node-js-driver-4-3-0-released/140479/2 and I've offered some potential solutions.
